packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -63,7 +63,7 @@ const AssetSummary = observer(() => {
     }
 
     // measure performance metrics
-    setPerformanceValue('login_time');
+    is_landing_company_loaded && setPerformanceValue('login_time');
```

Comment:
Hi @mitra-deriv 
We need to move this out from here and put inside the CFD section where we have  is_landing_company_loaded checks

Reply:
Yeah that's the point. We don't want the whole assets_summary to be loaded for this data, as users are going to different routes before seeing the data and our metrics is getting skewed. 

Let's have it with cfd section, even if the assets are not loaded we want to get this fired `setPerformanceValue('login_time');`